.github/workflows/dpf-version-test.yml

```
@@ -0,0 +1,440 @@
+          exit 1
+        }
+    
+    - name: Run service validation tests
+      run: |
+        source scripts/env.sh
+        export KUBECONFIG="${PWD}/kubeconfig"
+        
+        # Wait for services to be ready
+        echo "Waiting for DPU services to be ready..."
+        sleep 30
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)